02 - Community Expansions/02.01 Plugins by Category/Uncategorized plugins.md

```diff
@@ -73,4 +73,9 @@ Plugins which have not yet been categorized by the community.
 - [[obsidian-tts|Text to Speech]]
 - [[link-favicon|Link Favicons]]
 - [[get-info-plugin|Get Info]]
+- [[linked-data-helper|Linked Data Helper]]
+- [[linked-data-vocabularies|Linked Data Vocabularies]]
+- [[obsidian-regex-replace|Regex Find and Replace]]
+- [[obsidian-wordpress|Publish to WordPress for Obsidian]]
+- [[obsidian-icon-shortcodes|Icon Shortcodes]]
```

Comment:
It's interesting that only 5 lines were added here, even though 6 new plugin files were added.

The new matter.md plugin file has not been added here. I wonder why?

Reply:
Yeah, this is an interesting case indeed. I re-ran the script a couple of times and I still get the same result (this is one of those moments where it would help if it was sorted alphabetical, I think!)

Reply:
> The new matter.md plugin file has not been added here. I wonder why?

I logged this in #180 - and have submitted a PR to fix it in #183...

So I'm marking this conversation as resolved...